src/main/java/com/efficio/fieldbook/web/label/printing/controller/LabelPrintingController.java

```diff
@@ -261,16 +260,17 @@ public String showFieldmapLabelDetails(@ModelAttribute("labelPrintingForm") fina
 		form.setUserLabelPrinting(this.userLabelPrinting);
 
 		model.addAttribute(LabelPrintingController.AVAILABLE_FIELDS,
-				this.labelPrintingService.getAvailableLabelFieldsForFieldMap(true, locale));
+			this.labelPrintingService.getAvailableLabelFieldsForFieldMap(true, locale));
 
 		return super.show(model);
 	}
 
 	@RequestMapping(value = "/stock/{id}", method = RequestMethod.GET)
 	public String showStockListLabelDetails(@ModelAttribute("labelPrintingForm") final LabelPrintingForm form, final Model model,
```

Comment:
Is this still being called? **showStockListLabelDetails**

Reply:
It's not used anymore so we can delete it.